Assets/Scripts/KeyBindings/RestoreDefaults.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RestoreDefaults : MonoBehaviour
+{
+    public void Restore()
+    {
+        StoreKeyBindings.RestoreDefaults();
```

Comment:
This class has no use
It only adds one function layer above a call that could be made directly

Why make a whole script for this?

Reply:
because the other class is static and as such not compatible with a Unity Object, therefore the "vererbung" actually adds quite the functunality

Reply:
One could, of course, add that to any script creating a massive functionality overhead to many Unity objects